lib/utils/resolve-object.js

```diff
@@ -0,0 +1,12 @@
+module.exports = resolveObject
+
+function resolveObject (object) {
+  if (object._id) {
+    object.id = object._id
+    delete object._id
+
+    return object
+  } else if (object.id) {
+    return object
+  }
+}
```

Comment:
I think we can simplify this code a bit

```js
module.exports = resolveObject

function resolveObject (object) {
  if (object._id) {
    object.id = object._id
    delete object._id
  }

  return object
}
```

and maybe let's call the method `toObject` like we do here:
https://github.com/hoodiehq/pouchdb-hoodie-api/blob/master/utils/to-object.js (ignore the error handling and the extend in there)